streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignor.java

```diff
@@ -142,15 +142,6 @@ private void assignStandbyReplicaTasks(final TreeMap<UUID, ClientState> clientSt
         );
     }
 
-    // Visible for testing
-    static StandbyTaskAssignor createStandbyTaskAssignor(final AssignmentConfigs configs) {
```

Comment:
Refactored based on the previous PR comment https://github.com/apache/kafka/pull/10851#discussion_r817650626